package.json

```diff
@@ -26,9 +25,9 @@
     "eslint": "^5.16.0",
     "eslint-config-prettier": "^6.9.0",
     "eslint-plugin-prettier": "^3.1.2",
-    "husky": "^3.1.0",
+    "husky": "^4.2.3",
```

Comment:
You don't have to do this now but you should think about pinning the versions for your packages at some point. More info:
https://docs.renovatebot.com/dependency-pinning/

Reply:
Yes, but this was added by a command to setup `pre-commit`, and I wasn't using it before